tests/integration/Microsoft.Bot.ApplicationInsights.WebApi.Tests/ServiceResolutionTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Bot.ApplicationInsights.WebApi.Tests
+{
+        [TestClass]
+        [TestCategory("ApplicationInsights")]
+        public class ServiceResolutionTests
+        {
+            public ServiceResolutionTests()
+            {
+                // Arrange
+                //_server = new TestServer(new WebHostBuilder()
+                //.UseStartup<Startup>());
+                //_client = _server.CreateClient();
+            }
+            //[TestMethod]
```

Comment:
nice test :grinning:

Reply:
Yeah, strange dependency stuff.  There's no WebAPI tests atm..

Reply:
Added some WebAPI tests - scoped to small changes with ITelemetryInitializer